web/@linera/client/src/chain/mod.rs

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use futures::stream::StreamExt;
+use linera_base::identifiers::AccountOwner;
+use linera_client::chain_listener::ClientContext as _;
+use linera_core::{
+    client::ChainClient,
+    node::{ValidatorNode as _, ValidatorNodeProvider as _},
+};
+use serde::ser::Serialize as _;
+use wasm_bindgen::prelude::*;
+use web_sys::{js_sys, wasm_bindgen};
+
+use crate::{Client, Environment, JsResult};
+
+pub mod application;
+pub use application::Application;
+
+#[wasm_bindgen]
+pub struct Chain {
+    pub(crate) client: Client,
+    pub(crate) chain_client: ChainClient<Environment>,
+}
+
+#[derive(serde::Deserialize, tsify_next::Tsify)]
+#[tsify(from_wasm_abi)]
+pub struct TransferParams {
+    pub donor: Option<AccountOwner>,
```

Comment:
Why do you use _donor_? Never in my (almost) 10 years of experience (as a user and dev) have I seen this word. In 95% it's `from` and in the rest `source`.